### DISCUSSION TITLE
Welcome to AR-SaveWHU Discussions!

### DISCUSSION THREAD

**Discussion by hctian713** [Michael-Tian-Whu/AR-SaveWHU#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Luo-Z13** [comment]
How to install PIL?~~

**Reply by hctian713** [reply]
answer here

**Reply by Luo-Z13** [reply]
let me answer

**Reply by Luo-Z13** [reply]
> How to install PIL?~~

answer myself~

**Comment by hctian713** [comment]
pip install PIL
and wait to complete

**Reply by Luo-Z13** [reply]
Can u mark yourself?

**Reply by hctian713** [reply]
i dont think so
